src/mudlet-lua/lua/geyser/GeyserMiniConsole.lua

```diff
@@ -153,6 +153,81 @@ function Geyser.MiniConsole:insertLink(...)
   insertLink(self.name, ...)
 end
 
+--- inserts clickable text with right-click menu into the miniconsole at the end of the current line
```

Comment:
These will appear in the Geyser ldoc pages but don't give any real guidance on how to be used... but this is an issue with existing functions above too.

Reply:
They're largely wrappers around already documented stuff in the wiki, I could run through and include links to the wiki for the base functions they wrap

Reply:
That would be great.